spyder/plugins/shortcuts/widgets/summary.py

```diff
@@ -80,7 +82,7 @@ def __init__(self, parent=None):
         # group shortcuts by context
         shortcuts = groupby(sorted(CONF.iter_shortcuts()), key=itemgetter(0))
 
-        for context, group_shortcuts in shortcuts:
+        for _, group_shortcuts in shortcuts:
```

Comment:
```suggestion
        for __, group_shortcuts in shortcuts:
```

This is to avoid a clash with our translations function.